docs/File-Formats.md

```diff
@@ -1271,7 +1271,7 @@ Required fields:
 cancer_study_identifier: Same value as specified in study meta file
 genetic_alteration_type: TREATMENT_RESPONSE
 datatype: RESPONSE-VALUE
-stable_id: Any unique identifier within the study
+stable_id: Any unique identifier using a combination of alphanumeric characters and _
```

Comment:
please also add the validation to enforce it.

Reply:
Added.

Reply:
can't find it

Reply:
Sorry for being unclear. Uniqueness of the stable id is ensured by existing code. No change from my side was needed.

Reply:
Ok, I have added a check for the meta file stable_id in the data validator. A unit test for this was added to unit_tests_validate_data.py.